NatureArea.py

```diff
@@ -0,0 +1,245 @@
+                                                     url)
+
+    def set_labels(self):
+        """
+        Add labels, optionally in multiple languages.
+
+        A number of items have the word natur/domän etc. reservat
+        included in the name. In this case, the name is
+        added only in Swedish. Otherwise, if it's a pure
+        geographical name, it is added in a number of Latin script
+        languages.
+        """
+        swedish_name = self.raw_data["NAMN"]
+        if ("reservat" in swedish_name.lower() or
+                "nationalpark" in swedish_name.lower() or
+                "skärgård" in swedish_name.lower()):
+            languages = ["sv"]
+        else:
+            languages = [
+                "sv", "en", "da", "fi", "fr", "pt",
+                "pl", "nb", "nn", "nl", "de", "es"
+            ]
+        for language in languages:
+            self.add_label(language, swedish_name)
+
+    def set_descriptions(self):
+        """
+        Add descriptions in various languages.
+
+        National park format: "national park in Sweden".
+        Reserve format: "nature reserve in <län>, Sweden".
+        """
+        if self.raw_data["SKYDDSTYP"] == "Nationalpark":
+            np_dictionary = self.glossary["np_description"]
+            for language in np_dictionary:
+                description = np_dictionary[language]
+                self.add_description(language, description)
+        elif self.raw_data["SKYDDSTYP"] == "Naturreservat":
+            county_without_lan = self.raw_data["LAN"].split(" ")[:-1]
+            county_name = " ".join(county_without_lan)
+            if utils.get_last_char(county_name) == "s":
+                county_name = county_name[:-1]
+
+            nr_dictionary = self.glossary["nr_description"]
+            fi_locations = self.glossary["location_in"]["fi"]
+            ru_locations = self.glossary["location_in"]["ru"]
+            for language in nr_dictionary:
+                if language == "fi":
+                    description = nr_dictionary[language].format(
+                        fi_locations[county_name])
+                elif language == "ru":
+                    description = nr_dictionary[language].format(
+                        ru_locations[county_name])
+                else:
+                    description = nr_dictionary[language].format(county_name)
+                self.add_description(language, description)
+
+    def set_country(self):
+        """Set the country to Sweden."""
+        sweden = self.items["sweden"]
+        ref = self.make_stated_in_ref("Q29580583", "2009-12-09")
+        self.add_statement("country", sweden, ref=ref)
+
+    def set_iucn_status(self):
+        """Set the IUCN category of the area."""
+        #  To do: add no_value...
+        raw_status = self.raw_data["IUCNKAT"]
+        raw_timestamp = self.raw_data["URSBESLDAT"][1:11]
+        status_item = [x["item"] for
```

Comment:
Done. 0 is described as "novalue", but there should be a task higher up about implementing it properly.